Makefile.docker

```diff
@@ -16,7 +16,15 @@ DOCKER_CONTEXT := ${PROJECT_ROOT}
 
 DOCKER_USER_ARGS +=
 DOCKER_BUILD_ARGS += --build-arg PROJECT_VERSION=${PROJECT_VERSION}
-DOCKER_RUN_ARGS += -it --rm
```

Comment:
I would actually just remove `-it` from this line and add it to the `run-ubuntu-%` target.

Reply:
At least I don't think that test-ubuntu-% needs to have `-it` at all. We want to see the output, yes; but I can't think of any interaction that would need to be performed.

Reply:
Thank you for the suggestion! Makes sense. Done.